sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorLiveTestBase.cs

```diff
@@ -29,24 +27,14 @@ public MetricsAdvisorLiveTestBase(bool isAsync) : base(isAsync)
         internal const string AlertConfigurationId = "204a211a-c5f4-45f3-a30e-512fb25d1d2c";
         internal const string AlertId = "17571a77000";
         internal const string MetricId = "27e3015f-04fd-44ba-a20b-bc529a0aebae";
-        internal const string DataFeedId = "0072a752-1476-4cfa-8cf0-f226995201a0";
+        internal const string DataFeedId = "9860df01-e740-40ec-94a2-6351813552ba";
```

Comment:
I imagine this is used in other tests besides the Data feed ingestion ones. 
Will the change affect them too?

Reply:
The previous ID was only used in the old tests I deleted.

Other tests don't need this ID because:
- In Data Feed tests, we create a Data Feed from scratch and check that everything is being populated correctly. We don't need to use an existing Data Feed that already has data ingested.
- In other tests, even though we need the Data Feed to exist, we don't need its ID, but only the ID of one of its metrics (already available here as `MetricId`). The service is already able to identify to which Data Feed it corresponds.

Reply:
hmm then should this variable be moved to the `DataFeedIngestionLiveTests` ?

Reply:
You do have a point. Some other IDs could be moved to specific files as well. Personally, I prefer to keep them all in the same place. If, for any reason, we need to change the Data Feed or any of its components we're using for our tests, it would be easier if the IDs were all in the same place (a "base" configuration class) than if they were scattered around the test files. I think it keeps things more organized.